lib/generators/administrate/dashboard/dashboard_generator.rb

```diff
@@ -71,6 +71,21 @@ def field_type(attribute)
         end
       end
 
+      def column_type_for_attribute(attr)
+        type = klass.column_types[attr].type
+
+        if type == :integer && enum_type?(attr)
+          :string
```

Comment:
We can then add a new `enum: "Field::String",` entry to `ATTRIBUTE_TYPE_MAPPING`, so enums still get handled by the `String` field.

To turn off searching (until it's supported), we can change `ATTRIBUTE_OPTIONS_MAPPING` to:

```ruby
ATTRIBUTE_OPTIONS_MAPPING = {
  enum: { searchable: false },
  float: { decimals: 2 },
}
```